riptide-core/src/main/java/org/zalando/riptide/Route.java

```diff
@@ -115,4 +118,39 @@ static Route call(final ThrowingConsumer<ClientHttpResponse, ? extends Exception
         ThrowingConsumer<T, Exception> andThen(final ThrowingConsumer<R, ? extends Exception> consumer);
     }
 
+    static Route named(final Object name, final Route route) {
+        return new Route() {
+            @Override
+            public void execute(final ClientHttpResponse response, final MessageReader reader) throws Exception {
+                route.execute(response, reader);
+            }
+
+            @Override
+            public String toString() {
+                final ThrowingSupplier<String, Exception> stringer = () -> {
+                    final Method toString = name.getClass().getMethod("toString");
+                    return toString.getDeclaringClass() == Object.class ? "<code>" : name.toString();
```

Comment:
1. getDeclaringClass may return null
2. Why is it called on `toString` Method instead of `name` directly?